.github/arrange_release_directory.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I think this script should be relocated to a '/scripts' or '/release' folder. While I don't know if there are any strict conventions, the '.github' folder supported file types are:

- CODE_OF_CONDUCT.md
- CONTRIBUTING.md
- FUNDING.yml
- Issue and pull request templates and _config.yml_
- SECURITY.md
- SUPPORT.md

Reply:
moved to tools directory where the previous script used to be, and removed previous script.